### DISCUSSION TITLE
obsidian 是可以用一辈子不换的笔记软件吗

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#56] [备发布站的评论系统]
# obsidian 是可以用一辈子不换的笔记软件吗

  可以访问此文的双链版本，获得完整阅读体验：obsidian 是可以用一辈子不换的笔记软件吗 - 🌲 oldwinterの数字花园

https://notes.oldwinter.top/obsidian-%E6%98%AF%E5%8F%AF%E4%BB%A5%E7%94%A8%E4%B8%80%E8%BE%88%E5%AD%90%E4%B8%8D%E6%8D%A2%E7%9A%84%E7%AC%94%E8%AE%B0%E8%BD%AF%E4%BB%B6%E5%90%97

**Comment by bytemain** [comment]
这篇文章中的很多 `[[` 字段反而渲染出了一堆 html 代码，不符合预期

**Reply by oldwinter** [reply]
感谢纠错，看起来是不兼容`[[]]`这种语法嵌套了。好在我后面也几乎没用这种方式的语法。。。
